modules/modal/react/lib/ModalContent.tsx

```diff
@@ -212,10 +212,11 @@ const ModalContent = ({
   useAssistiveHideSiblings(modalRef);
 
   const content = (
-    <Container {...elemProps} ref={modalRef}>
+    <Container {...elemProps}>
       <CenteringContainer
+        ref={centeringRef}
         style={{width: windowSize.width % 2 === 1 ? 'calc(100vw - 1px)' : '100vw'}}
-        onClick={mergeCallback(onOverlayClick, elemProps.onClick)}
+        onMouseDown={onOverlayClick}
```

Comment:
Princess Popups close on `mousedown` and if we don't also use `mousedown`, they 2 systems will not work together very well. It is a slight change to how things close - `mousedown` vs `click`. Will this be a problem?

@jamesfan I know we talked about this before. I think you originally choose an item in the select on mousedown and we switched it to `click` instead...

Reply:
@NicholasBoll That's correct. It shouldn't be an issue to change it back to `mousedown`.